src/charts/chart.js

```diff
@@ -426,26 +334,37 @@ export default function Chart (_container) {
     if (!cache.root) {
       render()
     }
-    return eventCollector
+    return componentRegistry.getEvents()
```

Comment:
That's the way to get all events from sub-components

Reply:
✔️